web/src/lib/domain-logic/essentialQuestions.ts

```diff
@@ -179,4 +179,11 @@ export const EssentialQuestions: EssentialQuestion[] = [
     },
     fieldName: "employmentPlacementType",
   }),
+  // new EssentialQuestion({
+  //   shouldBeResetWhenIndustryChanges: true,
+  //   isQuestionApplicableToIndustry: (industry): boolean => {
+  //     return !!industry.industryOnboardingQuestions.isEmploymentAndPersonnelTypeApplicable;
+  //   },
+  //   fieldName: "todeletetype",
+  // }),
 ];
```

Comment:
For testing only: if you want to validate the test. Will be removed before a merge